decode/radio_frame.js

```diff
@@ -1,45 +1,43 @@
 var EthernetAddr = require('./ethernet_addr');
 var LogicalLinkControl = require('./llc_packet');
-
+var RadioBeaconFrame = require('./radio_beacon_frame');
 function RadioFrame() {
```

Comment:
It would be better to initialize the future members in the constructor. Check out how ipv4 does it for an example.